blog/2024-01-llms-graphs-part-2/index.md

```diff
@@ -277,11 +277,11 @@ for which I could not find much to read on is this:
 specialized methods)? 
 
 So could we ever dream of using LLMs to extract billions of triples from a large corpus of unstructured documents? Probably not
-if you're using OpenAI APIs, as it should be painfully expensive, or even if you're running 
-your own model, as it should be excruciatingly slow. So I am a bit pessimistic here. My instinct
+if you're using OpenAI APIs, as it would be painfully expensive, or even if you're running 
+your own model, as it would be excruciatingly slow. So I am a bit pessimistic here. My instinct
 is that you might be able to generate KGs from unstructured documents using the slightly older
 techniques like designing your own models or using extractor-based approaches like
-[DeepDive](http://deepdive.stanford.edu/).[^3] I know it's not exciting to not use LLMs, 
+[DeepDive](http://deepdive.stanford.edu/)[^3] and [SpanMarker](https://tomaarsen.github.io/SpanMarkerNER/)[^4]. I know it's not exciting to not use LLMs, 
```

Comment:
@semihsalihoglu-uw I added an additional citation to [SpanMarker](https://tomaarsen.github.io/SpanMarkerNER/) to the extractor-based approach section (to extract triples via named entities from unstructured data). I feel like the DeepDive project is no longer maintained, but SpanMarker is tightly integrated with the Hugging Face ecosystem, so it's worth a mention.

Reply:
Yes, DeepDive no longer exists but it was an example of a old and successful project for extracting structured records from unstructured documents (got acquired by Apple).